gdk/wizard/WizardData.py

```diff
@@ -0,0 +1,124 @@
+from gdk.wizard.ConfigEnum import ConfigEnum
+
+
+class WizardData:
+    def __init__(self, field_dict):
+        self.field_dict = field_dict
+        self.project_config = self.field_dict[ConfigEnum.COMPONENT.value.key]
+        self.component_name = next(iter(self.project_config))
+
+    def get_author(self):
+        try:
+            return self.project_config.get(
+                self.component_name, ConfigEnum.COMPONENT_NAME.value.default
+            )[ConfigEnum.AUTHOR.value.key]
```

Comment:
Use `dict.get(key,default)` instead of using default value after `KeyError` is raised.

Reply:
gotcha so remove all the try, except and replace it with dict.get right

Reply:
`self.project_config.get(self.component_name)` is going to be a dictionary object. Store it once and use it internally within the class.

Reply:
got it fixing right now